src/main/kotlin/org/seekerwing/aws/sqsconsumer/configuration/MessageProviderConfiguration.kt

```diff
@@ -20,6 +20,14 @@ import org.seekerwing.aws.sqsconsumer.model.Queue
  */
 data class MessageProviderConfiguration(
     val queue: Queue,
-    val messageFetcherConfiguration: MessageFetcherConfiguration = MessageFetcherConfiguration(),
+    val messageFetcherConfiguration: MessageFetcherConfiguration,
     val parallelism: Int = 1
-)
+) {
+
+    @Deprecated(
+        message = "Please specify messageFetcherConfiguration explicitly",
+        replaceWith = ReplaceWith("MessageProviderConfiguration(" +
+                "queue, /* TODO revisit parameter*/ MessageFetcherConfiguration(), parallelism)")
+    )
+    constructor(queue: Queue, parallelism: Int = 1) : this(queue, MessageFetcherConfiguration(), parallelism)
```

Comment:
same as above

Reply:
I actually think we should not increase major version, since there are not a lot of users yet.